lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -607,17 +627,23 @@ def construct_log_entry_in_rest_format(labels,
       )
     end
 
-    def write_request_via_grpc(entries:, log_name:, resource:, labels:)
+    def write_request_via_grpc(entries:,
```

Comment:
[Optional] If you rename this to just `write_request_grpc`, the header will fit on one line.

Reply:
Hmm... I found it easier to read with parameters on separate lines especially when some of them have default values.